stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/messages/header/MessageListHeader.kt

```diff
@@ -134,8 +135,8 @@ public fun DefaultMessageHeaderTitle(
     }
 
     val subtitle = when (messageMode) {
```

Comment:
This is a bit verbose unfortunately

Reply:
That's fine, the framework doesn't support quantity resource grabbing so we have to do it like this